pywrap.bzl

```diff
@@ -165,6 +172,79 @@ def pybind_extension(
         deps = ["%s" % cc_library_name],
     )
 
+def _generated_pywrap_win_def_file_impl(ctx):
+    cc_toolchain = find_cpp_toolchain(ctx)
+    feature_configuration = cc_common.configure_features(
+        ctx = ctx,
+        cc_toolchain = cc_toolchain,
+        requested_features = ctx.features,
+        unsupported_features = ctx.disabled_features,
+    )
+
+    dependency_libraries = []
+
+    pywrap_index = ctx.attr.pywrap_index
+    pywrap_infos = ctx.attr.dep[CollectedPywrapInfo].pywrap_infos.to_list()
+    if pywrap_index >= 0:
+        pywrap_infos = [pywrap_infos[pywrap_index]]
+
+    for pywrap_info in pywrap_infos:
+        cc_info = pywrap_info.cc_info
+
+        linker_inputs = cc_info.linking_context.linker_inputs.to_list()
+        
+        for linker_input in linker_inputs:
+            for lib in linker_input.libraries:
+                lib_copy = lib;
+                if not lib.alwayslink:
+                    lib_copy = cc_common.create_library_to_link(
```

Comment:
I don't think you need to create a lib_copy, the other rule does it to enforce alwayslink. This rule does not care about linking flags, so you can jus trely on "lib" itself.